en/access-dashboard.md

```diff
@@ -0,0 +1,148 @@
+---
+title: Access TiDB Dashboard
+summary: Learn how to access TiDB Dashboard in Kubernetes.
+category: how-to
+---
+
+# Access TiDB Dashboard
+
+TiDB Dashboard is a visualized tool in TiDB 4.0 used to monitor and diagnose the TiDB cluster. For details, see [TiDB Dashboard](https://github.com/pingcap-incubator/tidb-dashboard).
+
+This document describes how to access TiDB Dashboard in Kubernetes.
+
+## Quick start
+
+> **Note:**
+>
+> This guide shows how to quickly access TiDB Dashboard. Do not use this method in production.
+
+TiDB Dashboard is built-in the PD component in 4.0 versions. You can refer to the following example to quickly deploy a 4.0.0-rc TiDB cluster in Kubernetes.
```

Comment:
@Yisaer v4.0.0-rc is not the recommended TiDB version because some issues are found in this version. See 
![image](https://user-images.githubusercontent.com/41534398/82537857-548d8700-9b7d-11ea-9ad0-bc5be9ba0d09.png). What about using another version for your example?

```suggestion
TiDB Dashboard is built in the PD component in 4.0 versions. You can refer to the following example to quickly deploy a 4.0.0-rc TiDB cluster in Kubernetes.
```

Reply:
If this is just a quick start guide, not meant for the production environment, I think it's okay to use this version?